### PR TITLE
docs(tailwind-css): add `tailwind.config.ts` code block

### PR DESCRIPTION
**Pull request by Juneezee**
`create-next-app` has been generating `tailwind.config.ts` for TypeScript project since PR #47795. This PR adds a example code block for `tailwind.config.ts` file on the [Styling: Tailwind CSS page](https://nextjs.org/docs/app/building-your-application/styling/tailwind-css).

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 3f383e77cd4383f87dbf8e69693ab27a13217f07

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by leerob — APPROVED** [review]
Beautiful, thank you!

File: docs/02-app/01-building-your-application/05-styling/02-tailwind-css.mdx [hunk 1]
```diff
@@ -31,7 +31,27 @@ npx tailwindcss init -p
 
 Inside `tailwind.config.js`, add paths to the files that will use Tailwind CSS class names:
```
Comment (leerob) [inline docs/02-app/01-building-your-application/05-styling/02-tailwind-css.mdx:32]:
```suggestion
Inside your Tailwind configuration file, add paths to the files that will use Tailwind class names:
```

**Comment by leerob** [comment]
Please run Prettier 🙏

**Pull request merged**